libs/sdk-ui-filters/api/sdk-ui-filters.api.md

```diff
@@ -7,6 +7,7 @@
 import { AttributeFiltersOrPlaceholders } from '@gooddata/sdk-ui';
 import { DashboardDateFilterConfigMode } from '@gooddata/sdk-backend-spi';
 import { DateFilterGranularity } from '@gooddata/sdk-backend-spi';
```

Comment:
Seems there is some forgotten import from the SPI.

Reply:
This will be the next PR -> update of imports, need to write a script for it.